### PR TITLE
update module, fix travis errors correctly, and cleanup

### LINKED ISSUE #9: compile error
hello 
image

last rev azerothcore

### PR DESCRIPTION
**Pull request by milestorme**
This Pr fix’s build error and warnings , update module to skeleton and enable db_assembler and fix travis 
3 pull requests open that still don't fix what this PR fix's . closes https://github.com/azerothcore/mod-individual-xp/issues/9

### CODE REVIEW

**Comment by wizzymore** [comment]
So you combined more Pull Requests into one and did also some other changes that doesn't relate with the subject and is very hard to follow.

Please describe what the changes are for because, i repeat myself, it is very hard to follow what this PR is for

**Comment by milestorme** [comment]
No I merged my repo , It looks like others tryed to copy my work poorly , this commit is the correct cleanup of this module , why argue the other prs are no good . these fix’s have been here on my repo for longer than these other prs https://github.com/milestorme/Azerothcore--Modules/tree/master/mod-individual-XP

**Comment by milestorme** [comment]
Accept it or not , at least we have a clean working version of this on my repository so I can just give players mine if no merge 👍👍

**Comment by milestorme** [comment]
Also you might notice this is the only pr that passes travis

**Comment by pangolp** [comment]
I hope I didn't offend with my little contribution, but the code I modified to avoid the warning, I didn't copy it from anywhere, I was always trying ways to avoid the warning. @milestorme

**Comment by milestorme** [comment]
No @pangolp good try , it was @wizzymore my statement was directed he got all upset cause he couldn’t fix it himself  , have a look what I did and in the code change , that breaker was part of an if statement in the wrong place , it probably could be done better than I did it too , don’t give up bro , keep trying pr’s

**Comment by BarbzYHOOL** [comment]
3 PRs made into one is a bad habit, one fix/feature = one PR

I doubt Wizzy copied you, it's just a coincidence (we don't follow all the forks).


Alright so there are few issues as I can see:

- missing line breaks
- you removed some files from the .gitignore like ".patch" and some Microsoft visual studio file

That might be from another module that required a ".patch" though and in fact it makes me reconsider ignoring the ".patch" files by default

File: src/Individual_XP.cpp [hunk 1]
```diff
@@ -137,7 +145,7 @@ class Individual_XP_command : public CommandScript
         }
         else
         {
-            me->GetSession()->SendAreaTriggerMessage("Your current XP rate is %u", me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate);
+            me->GetSession()->SendAreaTriggerMessage("Your current XP multiplier is %u", me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate);
```
Comment (BarbzYHOOL) [inline src/Individual_XP.cpp:148]: why this change?

File: _config.yml [hunk 2]
```diff
@@ -1 +0,0 @@
-theme: jekyll-theme-slate
```
Comment (BarbzYHOOL) [inline _config.yml:1]: good to get rid of this xD

File: src/Individual_XP.cpp (hunk 1 above)
Comment (milestorme) [reply src/Individual_XP.cpp:148]: Because it’s a multiplier rather than an actual rate , for instance if you set rate In worldserver.conf to 2 thane set personal do to 2 your rate is 4x at least players. It’s correct English translation so as not to confuse players . I learned this from my players and made the change accordingly

**Comment by milestorme** [comment]
> 3 PRs made into one is a bad habit, one fix/feature = one PR
> 
> I doubt Wizzy copied you, it's just a coincidence (we don't follow all the forks).
> 
> Alright so there are few issues as I can see:
> 
> * missing line breaks
> * you removed some files from the .gitignore like ".patch" and some Microsoft visual studio file
> 
> That might be from another module that required a ".patch" though and in fact it makes me reconsider ignoring the ".patch" files by default

We shouldn’t merge the others they are no good 
I found if gitignore has ignore .patch we can’t commit patch’s to repo which is hard for modules that require a patch so global change is good I think

**Comment by Knindzagxg** [comment]
No need to point fingers at anyone, cause all AC members trying to help and provide more and more fixes daily.

It would be perfect, if you could create separate PR's and wait for Francesco, Brogga or others to fully check and approve changes 1 by 1.

In that way, we won't make any mistakes and it will be pretty easy to follow up new changes towards core, database or modules.

**Comment by milestorme** [comment]
No problem , all yours bro hook in aye , not gonna bother arguing with you guys , use wizzy mores Pr or one of the other broken prs I only submitted mine cause I’ve watched this module be left behind with multiple open broken prs but since you prefer to argue , I close and delete and you do this module yourself . ❤️❤️ Just copy my work like barbs did on mod-npc-services

**Comment by Knindzagxg** [comment]
I didn't meant to close your PR mate, nor i'm backing-up Wizzy.

Reason why i asked is cause i made that mistake few years ago, when i pulled few into one, and when i wanted to reverse it required a lot of work. It's simply to make things better, cause it really really helps when we add things 1 by 1 and in case there is some issue, we can check them and separate for future testing.

So, yet again, just make new PR's 1 by 1, and i'm pretty sure Francesco or Brogga will assist and check it pretty quickly.

**Comment by milestorme** [comment]
@Knindzagxg It was one pull request only mine the others had to be closed not merged , wow you can’t merge those others i never tryed to do that I don’t how barbs and you came to this conclusion , I had a fresh Pr with only good changes I never wanted the others merged together that would have corrupted the module , they needed to be closed . But I always get other devs shooting down my work here , I just don’t wanna help here anymore they get all offended and I’m just trying to help I want no modules nor special privileges like others do I just want EVERYONE to have the best modules out there on azerothcore without having to know about my repo I hate that I have to send people to my repo to get the cleaned stuff I want to send them here but everytime I try to help here I get smashed by another dev so I’m out bro I have people that appreciate me else where without being dug at by other devs I’m a soft person I hate this arguing and fighting I’m out , I don’t want this everyone I help it sucks and makes me cry 😢 I’m not a tough guy I just love to help people .

**Comment by Knindzagxg** [comment]
All should be good.

No reason to get emotional or anything at all, chill it out.
We are all trying to do whats best for AC, and that's why i'm asking you to open this one again and get issue sorted slowly.

As i mentioned few times before, some of those things are getting done slower, cause we are all doing something in our free time.

So yet again, chill out and open again in a day or w/e u wanna. Patience is the key, always.

**Comment by milestorme** [comment]
Yes I struggle bad with patience still , I like move fast so I can keep momentum. I like to run 🏃‍♂️

File: src/Individual_XP.cpp (hunk 1 above)
Comment (BarbzYHOOL) [reply src/Individual_XP.cpp:148]: ok interesting then why not

**Comment by BarbzYHOOL** [comment]
Also Wizzymore told me it's fine to merge this PR yesterday
